projects/media-viewer/src/lib/media-viewer.component.html

```diff
@@ -1,4 +1,6 @@
-<div id="outerContainer" [ngClass]="{ sidebarOpen: toolbarButtons.sidebarOpen | async }">
+<div id="outerContainer"
+     [ngClass]="{ sidebarOpen: toolbarButtons.sidebarOpen | async }"
+     [ngStyle]="{ width: width ? width : '100%' }">
```

Comment:
instead of doing this, you can just default the variable to '100%' when you declare it. but do it in the constructor rather than the actual declaration as otherwise it won't work for JUI because they use an old version of TypeScript